imports/ui/routes/methods.js

```diff
@@ -0,0 +1,6 @@
+const generateRoutes = (path, params) => {
+    path = path.slice(0, path.length - 4);
+    return path + params;
```

Comment:
What happens if you have this route '/user/:userId/post/:postId' ?
You have to generate your path for different routes. Don't stick to only one